qxf2/qxf2-lambdas
# Test comment reviewer lambda on localstack

Test holiday reminder lambda on LocalStack:
Test will do following:

* Deploy Lambda along with its dependencies and SQS
* Run test
* Delete Lambda and SQS queue

Locakstack Setup:

* cd to tests directory
* Start LocalStack using following command:
* `docker-compose up -d`

Check LocalStack status:

`localstack status`

Run Test:

* cd to comment_reviewer directory
* Install the dependencies - pip install -r requirements.txt
* cd to tests/test_comment_reviewer_lambda_on_localstack directory
* pytest -s -v test_comment_reviewer.py

Test ran successfully
![image](https://github.com/qxf2/qxf2-lambdas/assets/110374029/79a2f12c-d1e0-40d1-ad86-c8ca8cbfc2f1)
